TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateFactorPayload.swift

```diff
@@ -29,16 +29,16 @@ public protocol UpdateFactorPayload {
 public struct UpdatePushFactorPayload: UpdateFactorPayload {
   ///Factor Sid
   public let sid: String
-  ///Registration token generated by APNS when registering for remote notifications
-  public let pushToken: String
+  ///(Optional) Registration token generated by APNS when registering for remote notifications. Sending nil or empty will disable sending push notifications for challenges associated to this factor.
+  public let pushToken: String?
   
   /**
   Creates an **UpdatePushFactorPayload** with the given parameters
   - Parameters:
     - sid: Factor Sid
-    - pushToken: Registration token generated by APNS when registering for remote notifications
+    - pushToken: (Optional) Registration token generated by APNS when registering for remote notifications. Sending nil or empty will disable sending push notifications for challenges associated to this factor.
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/012ddda094c6c1e8c6d0425188ef9a322b4ad2d7/TwilioVerifySDK/TwilioVerify/Sources/Models/UpdateFactorPayload.swift#L39"></span>Line should be 195 characters or less: currently 210 characters<br />
<code>line_length</code> <code>UpdateFactorPayload.swift:39</code></td>
    </tr>
  </tbody>
</table>